src/app/sidebar/sidebar.tsx

```diff
@@ -242,6 +243,14 @@ class MainSideBar extends React.Component<{}, {}> {
                     </div>
                     <div className="middle hideScrollbarUntillHover">{this.getSessions()}</div>
                     <div className="bottom">
+                        <If condition = {clientData?.releaseinfo?.releaseavailable}>
```

Comment:
don't show this sidebar entry if users have turned off the "check for updates" in the settings

Reply:
does this make sense? what if they manually check for a new release?

Reply:
discussed offline, decided that showing the download link in the response for the `/releasecheck` command is sufficient